src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -224,6 +227,20 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
                       : ''}
                   </StructuredListCell>
                   <StructuredListCell>{lastTransitionTime}</StructuredListCell>
+                  <StructuredListCell>
+                    {isRunning(reason, status) && (
+                      <CancelButton
+                        type="Pipeline Run"
```

Comment:
should probably be 'PipelineRun' and 'TaskRun' (no space) for consistency with other places in the UI and to match the CRD kinds.